.tool/lint

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+for d in $(find . -type d -not -iwholename '*.git*' -a -not -iname '.tool' -a -not -iwholename '*vendor*'); do
```

Comment:
Why use not? If we only care about go files, I believe most of our go files will be in `cmd` and `pkg`.

How about `find . -type d -a \( -iwholename './pkg*' -o -iwholename './cmd*' \)`?